tests/templates/spark-mock-task-runner-job.yaml

```diff
@@ -0,0 +1,35 @@
+apiVersion: "sparkoperator.k8s.io/v1beta2"
+kind: SparkApplication
+metadata:
+  name: {{ .Name }}
+  namespace: {{ .Namespace }}
+spec:
+  type: Scala
+  mode: cluster
+  image: {{ .Image }}
+  imagePullPolicy: Always
+  mainClass: MockTaskRunner
+  mainApplicationFile: "https://infinity-artifacts.s3.amazonaws.com/scale-tests/dcos-spark-scala-tests-assembly-2.4.0-20190325.jar"
+  arguments:
+    - "1"
+    - "600"
```

Comment:
does it make sense to parameterize the arguments?

Reply:
I don't know. If this job will be reused in a new test with different arguments, it's fairly easy to refactor for parameterization in my opinion.

Reply:
makes sense, let's address that when it's needed